assets/js/Steps/Bibbox.spec.js

```diff
@@ -0,0 +1,1062 @@
+/**
```

Comment:
Most of the tests are located in the bibbox.spec. This is due to the way the state is implemented, and the components that depends on the machinestate cannot be tested without the machinestate, so they have to be tested through the Bibbox component